broker-util/oo-admin-repair

```diff
@@ -952,10 +952,15 @@ $chk_gear_mongo_node = true if fix_ssh_keys or fix_district_uids or fix_removed_
 $chk_district = true if fix_district_uids or fix_removed_nodes
 $chk_usage = true if fix_usage
 
-populate_user_hash
-populate_domain_hash
-populate_district_hash
-populate_app_hash
+# Skip populating hashes if --orphaned-envs is used
+unless fix_ssh_keys.nil? and fix_district_uids.nil? and fix_consumed_gears.nil? and
+   fix_removed_nodes.nil? and fix_usage.nil? and fix_gear_sizes.nil? and
+   !fix_orphaned_envs.nil?
+  populate_user_hash
+  populate_domain_hash
+  populate_district_hash
+  populate_app_hash
+end
```

Comment:
Not a big fan of this approach. If we add another flag/option to the tool, we'll have to remember to add it here as well. Instead, we should probably set a flag (either one for all data hashes or individually for different hashes) based on what options/flags are being passed. 

For starters, I'd vote for having a single boolean be used to determine whether all the data hashes need to be populated or not.